app/controllers/tweets_controller.rb

```diff
@@ -72,4 +74,10 @@ def set_tweet
     def tweet_params
       params.require(:tweet).permit(:content, :user_id)
     end
+
+    def check_user
+      if current_user!= @tweet.user
```

Comment:
Nice hax0r handling!

Is this condition ever true? i.e. can you ever hit the "Sorry" alert?